apiserver/clusterservice/clusterimpl.go

```diff
@@ -739,13 +739,14 @@ func CreateCluster(request *msgs.CreateClusterRequest, ns, pgouser string) msgs.
 			}
 		}
 
-		//verify that for autofail clusters we have a replica requested
+		// include a warning in the response indicating that autofail capabilities will not be able to be utilized
+		// until at least one replica is added to the cluster 
 		if newInstance.Spec.Replicas == "0" {
-			if request.AutofailFlag || apiserver.Pgo.Cluster.Autofail {
-				resp.Status.Code = msgs.Error
-				resp.Status.Msg = "replica-count of 1 or more is required when autofail is specified"
-				return resp
+			noReplicasResult := "Warning: Please add at least one replica to utilize autofail functionality"
+			if !request.AutofailFlag || apiserver.Pgo.Cluster.DisableAutofail {
```

Comment:
Just to clarify, the only thing that changed was the CLI, and not the API Server.  As for the old CLI option, I was hesitant to call it deprecated, since it is really gone/removed at this point.   Do you want me to add the old `autofail` CLI option back in and wire it up (also marking for deprecation), so the user can still use the old option in addition to the new `--disable-autofail` option?

Reply:
@andrewlecuyer I see my confusion. I would just leave this as is and ignore me.